grumpy-runtime-src/runtime/builtin_types.go

```diff
@@ -546,6 +546,18 @@ func builtinOrd(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
 	return NewInt(result).ToObject(), nil
 }
 
+func builtinPower(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	argc := len(args)
+	expectedTypes := []*Type{ObjectType, ObjectType}
+	if argc == 3 {
```

Comment:
you mean, `if argc = len(args)` ?

Reply:
no.
just using len(args), instead of declaring and initializing argc.

`func builtinPower(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
	expectedTypes := []*Type{ObjectType, ObjectType}
	if argc == len(args) {`

It's not a problem @alanjds , It's just for more performance.

Reply:
Done on 0a11050.
Waiting to merge.